content/en/agent/amazon_ecs/data_collected.md

```diff
@@ -17,24 +17,21 @@ further_reading:
 
 ### Metrics
 
-Amazon ECS metrics collected with the Datadog Agent:
+Amazon ECS on EC2 is a container management service for Docker containers running on EC2 instances. Metrics collected by the Agent when deployed in a Docker container are the same metrics collected by the Docker integration. Refer to the [Docker integration metrics][1] for a complete list of metrics.
```

Comment:
can we also mention that the Docker metrics will have these following four tags:
https://share.getcloudapp.com/4guJPEOZ

Reply:
just noticed the note below, let's just add the names of the tags please

Reply:
There are five tags in the screenshot, should I list all? Or just the first four?